app/App.js

```diff
@@ -404,15 +392,15 @@ export default class App extends Component {
                 type="button"
                 onClick={() => this.handleOpen(animation)}
               >
-                <div className={`cookie-icon ${animation ? `${animation}` : "default"} ${pulsing ? "pulsing" : ""}`} />
+                <div className={`xb-cookie-kit__cookie-icon ${animation ? `${animation}` : "default"} ${pulsing ? "xb-cookie-kit__pulsing" : ""}`} />
```

Comment:
Since we want to follow BEM all the way, even with its modifiers. We'll need to make the following change.
```suggestion
                <div className={`xb-cookie-kit__cookie-icon ${animation ? `xb-cookie-kit__cookie-icon--${animation}` : "xb-cookie-kit__cookie-icon--default"} ${pulsing ? "xb-cookie-kit__cookie-icon--pulsing" : ""}`} />
```

I know that the modifier classes seem to be overly verbose and that `xb-cookie-kit__cookie-icon default` is simpler than `xb-cookie-kit__cookie-icon xb-cookie-kit__cookie-icon--default`, but I think it would be better for the third-party developer experience to just know that we are using BEM naming conventions without any special exceptions.